tests/plugin.xml

```diff
@@ -26,6 +26,8 @@
     <name>Cordova Media Capture Plugin Tests</name>
     <license>Apache 2.0</license>
 
+    <dependency id="cordova-plugin-media" />
```

Comment:
It appeared during the testing and prevents one manual test from running - it's failing with `Media` undefined error while trying to play a recorded file.